cli/cmd/customer_update.go

```diff
@@ -62,28 +67,49 @@ func (r *runners) updateCustomer(cmd *cobra.Command, _ []string) (err error) {
 		r.args.customerUpdateType = "prod"
 	}
 
-	channelID := ""
-	if r.args.customerUpdateChannel != "" {
+	channels := []kotsclient.CustomerChannel{}
```

Comment:
This should validate somewhere above that if multiple channels are specified, then `customerCreateDefaultChannel` must have a value.

Reply:
I was going for:

You can specify which is default, but if you don't specify, the first is the default.  I included that in the help text.